applylib/applyset/health.go

```diff
@@ -17,105 +17,34 @@ limitations under the License.
 package applyset
 
 import (
-	"encoding/json"
 	"strings"
 
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/klog/v2"
+	"sigs.k8s.io/kustomize/kstatus/status"
 )
 
 // isHealthy reports whether the object should be considered "healthy"
 // TODO: Replace with kstatus library
 func isHealthy(u *unstructured.Unstructured) bool {
-	// Check if the resource is scheduled for deletion
-	deletionTimestamp := u.GetDeletionTimestamp()
-	if deletionTimestamp != nil {
-		klog.Infof("object %s is scheduled for deletion", humanName(u))
+	result, err := status.Compute(u)
```

Comment:
`status.Compute` contains the logics from the deleted codes (deletionTimestamp, no-condition k8s resources and legacy status.conditions field).